.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
```

Comment:
[Optional] You seem to have used the incorrect linters configuration for this project. Since it's a HTML + CSS + JS project, [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) would've been the proper configuration to use. I suggest that you modify it and take note in the case of subsequent projects.